### PR TITLE
Comment out ellipsis in code blocks

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Ellipsis in markdown code blocks are syntax errors.

In order to have smooth transition to Prettier and ESLint the PR comments out such bare ellipses. Refer the https://github.com/mdn/content/pull/18171#issuecomment-1179670484 for more context.

### CODE REVIEW

File: files/en-us/web/javascript/reference/functions/arrow_functions/index.md [hunk 1]
```diff
@@ -199,8 +199,8 @@ const obj = { // does not create a new scope
   },
 }
 
-obj.b(); // prints undefined, Window {...} (or the global object)
-obj.c(); // prints 10, Object {...}
+obj.b(); // prints undefined, Window { } (or the global object)
+obj.c(); // prints 10, Object { }
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/functions/arrow_functions/index.md:203]:
```suggestion
obj.b(); // prints undefined, Window { /* … */ } (or the global object)
obj.c(); // prints 10, Object { /* … */ }
```

File: files/en-us/web/javascript/reference/functions/arrow_functions/index.md [hunk 2]
```diff
@@ -215,7 +215,7 @@ const obj = {
 
 Object.defineProperty(obj, 'b', {
   get: () => {
-    console.log(this.a, typeof this.a, this); // undefined 'undefined' Window {...} (or the global object)
+    console.log(this.a, typeof this.a, this); // undefined 'undefined' Window { } (or the global object)
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/functions/arrow_functions/index.md:218]:
```suggestion
    console.log(this.a, typeof this.a, this); // undefined 'undefined' Window { /* … */ } (or the global object)
```

File: files/en-us/web/javascript/reference/functions/get/index.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ Note the following when working with the `get` syntax:
 
   ```js example-bad
   {
-    x: ..., get x() { }
+    x: …, get x() { }
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/functions/get/index.md:64]:
```suggestion
    x: /* … */, get x() { /* … */ }
```

File: files/en-us/web/javascript/reference/functions/set/index.md [hunk 4]
```diff
@@ -51,7 +51,7 @@ Note the following when working with the `set` syntax:
 - It must not appear in an object literal with another `set` or with a
   data entry for the same property.
   ( `{ set x(v) { }, set x(v) { } }` and
-  `{ x: ..., set x(v) { } }` are forbidden )
+  `{ x: …, set x(v) { } }` are forbidden )
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/functions/set/index.md:54]:
```suggestion
  `{ x: /* … */ , set x(v) { /* … */ } }` are forbidden )
```

File: files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md [hunk 5]
```diff
@@ -90,7 +90,7 @@ object or a custom object.
 ```js
 function logIterable(it) {
   if (!(Symbol.iterator in it)) {
-    console.log(it, ' is not an iterable object...');
+    console.log(it, ' is not an iterable object…');
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md:93]:
```suggestion
    console.error(`${it} is not an iterable object.');
```

File: files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md [hunk 6]
```diff
@@ -113,7 +113,7 @@ logIterable('abc');
 // c
 
 logIterable(123);
-// 123 is not an iterable object...
+// 123 is not an iterable object…
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md:116]:
```suggestion
// 123 is not an iterable object.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/javascript/reference/functions/set/index.md (hunk 4 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/functions/set/index.md:54]: `/* … */` indicates optionalness, but here `…` means it _has to_ be substituted with a value

**Comment by Josh-Cena** [comment]
@teoli2003 @OnkarRuikar Sorry I was not closely following this process before. I have a question about semantics.

The goal of this update is to make code blocks syntactically valid. However some changes don't really make the code any more valid. For example:

```diff
- const a = ...;
+ const a = /* … */;
```

Instead, I think it actually makes the point less clear. What about using `…` without enclosing it in comments in places where we deliberately indicate "omission" instead of "optional content"? For example, "this object will be logged in console as `{ … }` looks better than `{ /* … */ }`, because there's indeed no comments logged!

**Comment by Josh-Cena** [comment]
(Also, I'm skeptical about replacing `...` with `…` in code blocks, though I don't have strong opinions. It doesn't read great with a monospace font.)

File: files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md (hunk 5 above)
Comment (teoli2003) [reply files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md:93]: So let's keep it the original way?

Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md:93]: Yeah

**Comment by OnkarRuikar** [comment]
> const a = ...;

Are we going to lint inline code(e.g. \`const a = 9\` as well? 
* If no, then `const a = ...;` won't matter as long as it conveys the meaning.
* If yes, and for code blocks ```` ```js ````, what should be the replacement?

***

```diff
- console.log(foo); // { … }
+ console.log(foo); // { /* … */ }
```

I got these as review suggestions. You guys decide and, if needed, I'll correct it in all the pages.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**